tensorboard/plugins/distributions/tf_distribution_dashboard/tf-distribution-dashboard.html

```diff
@@ -71,61 +70,117 @@
       </div>
 
       <div class="center">
-        <tf-panes-helper
-          categories="[[_categories]]"
-          data-type="[[dataType]]"
-          data-provider="[[dataProvider]]"
-          data-not-found="[[dataNotFound]]"
-          run2tag="[[run2tag]]"
-          selected-runs="[[_selectedRuns]]"
-          repeat-for-runs
+        <template is="dom-if" if="[[_dataNotFound]]">
```

Comment:
I think we will wind up removing these all once we have logic to only display active plugins.
If we were to keep them, we would probably want to at least refactor out the standard "here's the TensorBoard github, here's the readme, ...." part.